### PR TITLE
 adding api-gateway server

### PR DESCRIPTION
**Pull request by bllexe**
Could you please check this when you are available.

### CODE REVIEW

File: api-gateway/src/main/java/dev/tigris/apiGateway/controller/HystrixController.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package dev.tigris.apiGateway.controller;
+
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/fb")
```
Comment (devTigris) [inline api-gateway/src/main/java/dev/tigris/apiGateway/controller/HystrixController.java:8]: might be more readable endpoint

**Review by devTigris — COMMENTED** [review]
good jobs.

**Review by devTigris — APPROVED** [review]
good jobs.

**Comment by bllexe** [comment]
completed before k8s deployment.

**Pull request merged**